pages/dkp/konvoy/2.1/choose-infrastructure/airgapbundle/index.md

```diff
@@ -0,0 +1,510 @@
+---
+layout: layout.pug
+navigationTitle: Air-Gapped Bundles
+title: Air-Gapped Bundles
+menuWeight: 10
+excerpt: Download and install air-gapped bundles when provisioning in an air-gapped environment.
+beta: false
+enterprise: false
+---
+
+## Review the air-gapped bundle contents
+
+The air-gapped bundle provided contains the necessary OS dependencies and images required to set up a DKP Essential installation in an air-gapped environment. The download and installation of these bundles is agnostic to a specific environment, and is compatible wiht any of the environments listed in the table below:
+
+| Operating System      | Kernel                      | Air-Gapped     |
+|-----------------------|-----------------------------|----------------|
+| [CentOS 7.9][centos7] | 3.10.0-1160.el7.x86_64      | Yes            |
+| [RHEL 7.9][rhel_7_9]  | 3.10.0-1160.el7.x86_64      | Yes            |
+
+The air-gapped bundle includes the following packages and tooling:
+
+- Command line tooling
+- Konvoy air-gapped bundle,
+- Kommander air-gapped bundle,
+- Konvoy Image Builder
+- CentOS/RHEL 7.9 dependencies
+- Setup script
+- Cluster initialization script
+- Docker registry image
+
+## Download and expand  the air-gapped bundle
+
+
+1.  Define an environment variable for the Kubernetes version that corresponds with Konvoy release you are installing. You can find the correct Kubernetes version by checking the release notes for the release you are installing.
+
+    ```sh
+    export VERSION=1.21.6
+    ```
+
+1.  Download the air-gapped bundle files.
+
+    ```sh
+    curl -O downloads.d2iq.com/dkp/v2.1.1/dkp_airgapped_bundle_v2.1.1_linux_amd64.tar.gz
+    curl -O downloads.d2iq.com/konvoy/airgapped/pip-packages.tar.gz
+    ```
```

Comment:
I'm confused with the version numbering/naming here.   In the old version of these instructions ( https://docs.d2iq.com/dkp/konvoy/2.1/choose-infrastructure/awsgovcloud/prerequisites/) the air-gapped bundle name included the Kubernetes version string, and DID NOT include the DKP version string.

Since this is for 2.1.1, it would surprise me if we now have published different packages.   Also, I tried to download the file you mention here, and it is not found.   Did engineering repackage all this stuff with this scheme?

At any rate, the names here are critical here, so this needs to get sorted before this can be merged.   

```suggestion
1.  Download the air-gapped bundle files.

    ```sh
    curl -O downloads.d2iq.com/konvoy/airgapped/os-packages_$VERSION_x86_64_rpms.tar.gz
    curl -O downloads.d2iq.com/konvoy/airgapped/pip-packages.tar.gz
    ```
```
(this change will also impact the other places on this page where this change was made, obviously)

Reply:
We should continue to publish the old package names in 2.1 docs. In 2.2, we can update these if that's the right thing.